docs/personas.md

```diff
@@ -0,0 +1,31 @@
+# Personas
+## [P1] <a name="P1">Propietario</a>
+Dueño y CEO del restaurante. Como propietario es el encargado de tomar decisiones acertadas para el futuro de su negocio, además de gestionar el personal. 
+
+## [P2] <a name="P2">Camarero</a>
+Un empleado del restaurante. Encargado de atender de forma a las mesas, servir la comida hecha por el [cocinero](#P3) y del trato con los [clientes](#P4).
+
+## [P3] <a name="P3">Cocinero</a>
+Empleado del restaurante encargado de hacer las comidas que piden los [clientes](#P4).
+
+## [P4] <a name="P4">Cliente</a>
+Consumidor que haya pasado por el restaurante. Se encarga de registrar una opinión nueva en el sistema.
+
+---
+
+# Historias de Usuario
+* **[HU1](https://github.com/JAntonioVR/IV-2021-2022/issues/2)**: Como [propietario](#P1), quiero saber la opinión que tiene mi clientela sobre algún camarero concreto en un rango de fechas determinado.
+* **[HU2](https://github.com/JAntonioVR/IV-2021-2022/issues/3)**: Como [propietario](#P1), quiero saber la opinión que tiene mi clientela sobre la comida que servimos en un rango de fechas determinado.
```

Comment:
Todas las historias de usuario deben incluir "Dado que". Por favor, tratad de entender el concepto con el material de la asignatura o como veáis antes de lanzaros a escribir. Esta
1. No dice en base a qué
2. Es simplemente búsqueda.
3. No está relacionada con la lógica de negocio (las *estrategias* de las que hablabas)
4. Es una tarea de UI.